ister_test.py

```diff
@@ -55,6 +55,7 @@
 import netifaces
 
 import ister
+import ister_gui
```

Comment:
I want to have the static network detection and static file writing logic tested here most of all I think, wrap the open call with the decorator to mock it.